runners/core-java/src/main/java/org/apache/beam/runners/core/GBKIntoKeyedWorkItems.java

```diff
@@ -49,7 +49,7 @@
         input.getWindowingStrategy().getWindowFn().windowCoder());
     PCollection<KeyedWorkItem<KeyT, InputT>> collection = PCollection.createPrimitiveOutputInternal(
```

Comment:
I think I'll leave it. Trying to inline the collection requires using `setCoder` which then requires specifying the type for the `createPrimitiveOutputInternal` which makes the formatting much worse.